.github/workflows/pull-request.yml

```
@@ -0,0 +1,158 @@
+                | jq -r '.[0].comment')" >> $GITHUB_OUTPUT    
+  stage_rollback:
+   needs: [manual_approval_rollback, stage_deploy]
+   if: contains(needs.manual_approval_rollback.outputs.comment, 'approve')
+   runs-on: self-hosted
+   steps:
+    - name: Rollback tag
+      run: echo "test_val_2=${GITHUB_REF##*/}.$((RUN_NUM - 1))"
+    - name: Rollback tags
+      run: echo ${{ needs.stage_deploy.outputs.backend_tag }}
+    - name: Checkout
+      uses: actions/checkout@v2
```

actionlint (1.7.9)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)